components/Dashboard/index.tsx

```diff
@@ -60,11 +58,11 @@ function Component() {
         }
     }
 
-    // useEffect(() => {
-    //     if (loginInfo.zilAddr !== null) {
-    //         updateShowZilpay(false)
-    //     }
-    // }, [loginInfo.zilAddr])
+    useEffect(() => {
+        if (loginInfo.zilAddr !== null) {
+            updateShowZilpay(false)
```

Comment:
this is not handling the wallet switch properly

Reply:
Hi @tralkan wallet switch working now after removing network condition(if network !== loginInfo.net). I just post comment on notion

Reply:
what if loginInfo.zilAddr != null but still a switch? not updating here